README.md

```diff
@@ -135,3 +135,10 @@ Some knn-related tasks are included in the main benchmarks. If you specifically
 KNN/HNSW there is a script dedicated to that in src/python/knnPerfTest.py which has instructions on
 how to run it in its comments.
 
+## Testing with higher dimension vectors
+
+By default we use 100 dimension vectors, to use higher dimension vectors, you need to:
+
+1. run `src/python/infer_token_vectors.py` to get `xxx.vec` and `xxx.tok` file, also do not forget to update the model you want to use. E.g. for 768 dimensions you need `enwiki-20120502-mpnet.vec` and `enwiki-20120502-mpnet.tok` as output file and you need to set the model like `model = SentenceTransformer('all-mpnet-base-v2')` in `infer_token_vectors.py` (which is already there)
```

Comment:
maybe "set the  model ... by editing infer_token_vectors.py. The supported models are listed there in comments.